cpp/src/arrow/compute/kernels/hash_aggregate_test.cc

```diff
@@ -1917,57 +2243,7 @@ TEST(GroupBy, MinMaxBinary) {
   }
 }
 
-TEST(GroupBy, MinMaxFixedSizeBinary) {
```

Comment:
where did this go?

Reply:
I'm not sure. There is a test case of the same name, but with different code, just above in the diff. Both these test-cases I did not author. I'll defer to @westonpace for which of these two to keep.

Reply:
If you do not author both - can we not remove it then?

Reply:
The original test is called "MinMaxBinary" and it seems you renamed it to "MinMaxFixedSizeBinary" and deleted the original "MinMaxFixedSizeBinary" test

Reply:
> can we not remove it then?

I'm asking @westonpace to help figure out whether we need one or both.

> The original test above is called "MinMaxBinary" and it seems you renamed it to "MinMaxFixedSizeBinary" and deleted the original "MinMaxFixedSizeBinary" test

The diff in this PR doesn't appear consistent with the current version in the main branch. I don't know what should be the conflict resolution here and so I'm asking @westonpace to help figure it out.

Reply:
Looks like https://github.com/apache/arrow/blob/main/cpp/src/arrow/compute/kernels/hash_aggregate_test.cc#L1869 exists in main branch but you removed the test in this PR - can you restore it?

Reply:
Will do. As noted elsewhere, I'd like @westonpace to review this conflict resolution.

Reply:
LGTM now